### DISCUSSION TITLE
ReST topic titles appear unstyled.

### DISCUSSION THREAD

**Discussion by cjw296** [pradyunsg/furo#291] [Potential Issue] [answered]
Consider this ReST:

```
Methods of replacement
----------------------

There are three different methods of mocking out
functionality that can be used to replace functions, classes
or even individual methods on a class. Consider the following module:

.. topic:: testfixtures.tests.sample1
   :class: module

   .. literalinclude:: ../testfixtures/tests/sample1.py
      :pyobject: X
```
With the  RTD theme, this looks like:
<img width="727" alt="Screenshot 2021-11-03 at 06 07 44" src="https://user-images.githubusercontent.com/632049/140016210-86e4a011-e340-4c0f-b910-772c8a4cc811.png">

However, with Furo, it appears unstyled:
<img width="843" alt="Screenshot 2021-11-03 at 06 08 01" src="https://user-images.githubusercontent.com/632049/140016207-f4e9c0d5-94f2-45c3-84d6-fa8117772e70.png">

If asked, I'd have a preference for matching the `font-weight: bold` style of the RTD theme and with reduced padding below it, to link it to the topic body, but in any case I'd hope it would be differentiated from normal body text.

**Comment by pradyunsg** [comment, marked as answer]
Duplicate of https://github.com/pradyunsg/furo/discussions/256
